liferay-frontend/liferay-portal
# LPS-122218 Add new automation test asserting a new remote app can be added and listed in a table on admin page

Please backport to 7.3.x

### JIRA Ticket:
https://issues.liferay.com/browse/LPS-122218

### Description:
This adds an automated functional test to validate a Remote App can be added and listed in a table on admin page

#### TestCase: Add a Remote App
1. Navigate to App Menu > Remote App
2. Click plus button
3. Enter text in Name field: Test Remote App
4. Enter text in URL field: http://www.liferay.com
5. Click Save button
6. Verify success message
7. Assert created remote app name listed in table: Test Remote App
8. Assert URL listed in table: http://www.liferay.com
